frontend/src/pages/AccountReceive.vue

```diff
@@ -11,7 +11,19 @@
       </div>
     </div>
 
-    <div v-else class="q-mx-auto" style="max-width: 800px">
+    <div
+      v-else-if="!isAccountSetup && !advancedMode"
+      class="dark-toggle text-center text-bold q-pa-md q-mb-lg"
+      style="border-radius: 15px"
+      :style="isDark ? 'color: #FFFAEB; background-color: #7C5E10' : 'color: #513C06; background-color: #FCEFC7'"
```

Comment:
It is better to use one of the quasar shading classes or the sass variables we have setup in the app. It allows us to keep the colors consistent in the app.

Reply:
Yea try to avoid `style` as much as possible for the reason @alexkeating mentioned. I believe quasar has classes for border radius, maybe max width also

Reply:
Good to know, thank you! Addressed it in https://github.com/ScopeLift/umbra-protocol/pull/571/commits/d34d75c3e854848b58237035fb4eea4868304fdc
- Quasar Palette didn't really have colors that were close so created custom text and bg colors in sass file
- Quasar rounded border class wasn't round enough so also created a custom class in sass
- Quasar max-width option didn't match our widths, so just reused an existing sass class.

Reply:
Ran prettier on .sass https://github.com/ScopeLift/umbra-protocol/pull/571/commits/cdbf1a522403bc910a36c40d0bf0b750e7e392ea

Reply:
Thanks! Since you pulled things out into classes, should we also pull out existing hardcoded classes, e.g. the `:style="isDark ? 'color: #FFFAEB; background-color: #7C5E10' : 'color: #513C06; background-color: #FCEFC7'"` line in this PR? Or a follow up PR?

Reply:
There seems to be a lot more of those. Let me create an issue for it and fix them in a separate PR

Reply:
Created an issue https://github.com/ScopeLift/umbra-protocol/issues/572